ddtrace/debugging/_exception/auto_instrument.py

```diff
@@ -0,0 +1,102 @@
+from itertools import count
+import sys
+from threading import current_thread
+from types import FrameType
+import typing as t
+import uuid
+
+import attr
+
+from ddtrace.debugging._probe.model import LiteralTemplateSegment
+from ddtrace.debugging._probe.model import LogLineProbe
+from ddtrace.debugging._signal.collector import SignalCollector
+from ddtrace.debugging._signal.snapshot import DEFAULT_CAPTURE_LIMITS
+from ddtrace.debugging._signal.snapshot import Snapshot
+from ddtrace.internal.processor import SpanProcessor
+from ddtrace.span import Span
+
+
+@attr.s
+class SpanExceptionProbe(LogLineProbe):
+    @classmethod
+    def build(cls, exc_id, frame):
+        # type: (uuid.UUID, FrameType) -> SpanExceptionProbe
+        _exc_id = str(exc_id)
+        message = "Exception Debugging (exception %s)" % exc_id
```

Comment:
I would add frame details like file/line/method into the message, maybe logs much more human readable. once it move to private track, message would not be needed anymore